### DISCUSSION TITLE
What to do at night?

### DISCUSSION THREAD

**Discussion by addiejanssen** [addiejanssen/domoticz-solaredge-modbustcp-plugin#14] [Q&A]
The plugin continues to retrieve information from the inverter while the inverter is not producing power.
Perhaps it is a better idea to implement a check on the inverter status and based on that process the inverter values.

Let's investigate options and if they make sense.

**Comment by mcmellow** [comment]
Thanks for opening this issue.

**Comment by Frank18000** [comment]
i think looking at the status is the best way forward
